### PR TITLE
Add DCS-7358 series

### PR DESCRIPTION
**Pull request by colinmacgiolla**
Small PR to add the DCS-7358 chassis, and a linecard

### CODE REVIEW

File: device-types/Arista/DCS-7358.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    maximum_draw: 2000
+  - name: PS2
+    type: iec-60320-c20
+    maximum_draw: 2000
+  - name: PS3
+    type: iec-60320-c20
+    maximum_draw: 2000
+  - name: PS4
+    type: iec-60320-c20
+    maximum_draw: 2000
```
Comment (danner26) [inline device-types/Arista/DCS-7358.yaml:21]: Are these PSU's modular? If so, then they should be modelled here under `module-bays` and the associated PSU's should be created in the module-types folder if they do not already exist

File: device-types/Arista/DCS-7358.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
+---
+manufacturer: Arista
+model: DCS-7358
+slug: arista-dcs-7358
+part_number: DCS-7358
+u_height: 4
+is_full_depth: false
```
Comment (danner26) [inline device-types/Arista/DCS-7358.yaml:7]: Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.

1. Please add the `airflow` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not
2. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

If you have any questions, please let us know :)

**Review by danner26 — CHANGES_REQUESTED** [review]

File: device-types/Arista/DCS-7358.yaml (hunk 2 above)
Comment (colinmacgiolla) [reply device-types/Arista/DCS-7358.yaml:7]: Is it better to have the chassis only weight, or the fully loaded weight?

Comment (danner26) [reply device-types/Arista/DCS-7358.yaml:7]: Correct, it should be the chassis only weight

**Comment by colinmacgiolla** [comment]
Should be good now, but no idea how to mark the requested change as done 😉

**Pull request closed without merging**